### ISSUE TITLE
CAS6 - configured to accept a static list of credentials for authentication

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#3789]
cas_1                                | CAS is configured to accept a static list of credentials for authentication. While this is generally useful for demo purposes, it is STRONGLY recommended that you DISABLE this authentication method by setting 'cas.authn.accept.enabled=false' and switch to a mode that is more suitable for production.>

**Comment by landryb** [comment]
those warnings just next to this one are also ugly, "something" should be done to generate by-instance keys:
```
<Secret key for encryption is not defined for [OAuth JWT Access Tokens]; CAS will attempt to auto-generate the encryption key>
<Generated encryption key [xxx] of size [256] for [OAuth JWT Access Tokens].
The generated key MUST be added to CAS settings under setting [cas.authn.oauth.access-token.crypto.encryption.key].>
<Secret key for signing is not defined for [OAuth JWT Access Tokens]. CAS will attempt to auto-generate the signing key>
<Generated signing key [xxx] of size [512] for [OAuth JWT Access Tokens].
The generated key MUST be added to CAS settings under setting [cas.authn.oauth.access-token.crypto.signing.key].>
<Secret key for encryption is not defined for [OAuth Registered Service]; CAS will attempt to auto-generate the encryption key>
<Generated encryption key [xxx] of size [256] for [OAuth Registered Service].
The generated key MUST be added to CAS settings under setting [cas.authn.oauth.crypto.encryption.key].>
<Secret key for signing is not defined for [OAuth Registered Service]. CAS will attempt to auto-generate the signing key>
<Generated signing key [xxx] of size [512] for [OAuth Registered Service].
The generated key MUST be added to CAS settings under setting [cas.authn.oauth.crypto.signing.key].>
<Secret key for encryption is not defined for [Ticket-granting Cookie]; CAS will attempt to auto-generate the encryption key>
<Generated encryption key [xxx] of size [256] for [Ticket-granting Cookie].
The generated key MUST be added to CAS settings under setting [cas.tgc.crypto.encryption.key].>
<Secret key for signing is not defined for [Ticket-granting Cookie]. CAS will attempt to auto-generate the signing key>
<Generated signing key [xxxx] of size [512] for [Ticket-granting Cookie].
The generated key MUST be added to CAS settings under setting [cas.tgc.crypto.signing.key].>
<Secret key for signing is not defined under [cas.webflow.crypto.signing.key]. CAS will attempt to auto-generate the signing key>
<Generated signing key [xxx] of size [512].
The generated key MUST be added to CAS settings under setting [cas.webflow.crypto.signing.key].>
<Secret key for encryption is not defined under [cas.webflow.crypto.encryption.key]. CAS will attempt to auto-generate the encryption key>
<Generated encryption key [xxx] of size [16].
The generated key MUST be added to CAS settings under setting [cas.webflow.crypto.encryption.key].>
```

**Comment by fvanderbiest** [comment]
Yep, clearly our integration of CAS6 is not "done".

**Comment by arnaud-morvan** [comment]
Done by @fvanderbiest this morning

**Comment by landryb** [comment]
first part fixed by georchestra/datadir#249 - second part unfixed, there will still be warnings about default keys being generated at runtime.
